src/bundle.c

```diff
@@ -348,7 +348,6 @@ int add_subscriptions(struct list *bundles, int current_version, struct manifest
 
 		file = search_bundle_in_manifest(mom, bundle);
 		if (!file) {
-			printf("%s bundle name is invalid, skipping it...\n", bundle);
```

Comment:
I would prefer to keep this printf, since it is useful for 'bundle-add', and shouldn't ever occur for 'update'.